ingestion/setup.py

```diff
@@ -120,7 +120,7 @@ def get_long_description():
 plugins: Dict[str, Set[str]] = {
     "airflow": {VERSIONS["airflow"]},  # Same as ingestion container. For development.
     "amundsen": {VERSIONS["neo4j"]},
-    "athena": {"PyAthena[SQLAlchemy]"},
+    "athena": {"pyathena==2.25.2"},
```

Comment:
Latest version of [pyathena==3.0.6](https://pypi.org/project/pyathena/) removed support for python 3.7 [ref](https://pypi.org/project/pyathena/#toc-entry-1)

We will fix it now to a lower version till OM has support for python 3.7

Also this after installing the latest pyathena version this [import](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/database/athena/metadata.py#L16) needs to be changed

Reply:
thanks, added a comment [here](https://github.com/open-metadata/OpenMetadata/issues/12082) to track this work